src/Sentry/Laravel/Tracing/Middleware.php

```diff
@@ -46,6 +53,13 @@ public function handle($request, Closure $next)
     public function terminate($request, $response): void
     {
         if ($this->transaction !== null && app()->bound('sentry')) {
+            if ($this->appSpan !== null) {
+                $this->appSpan->finish();
+            }
+
+            // Make sure we set the span in the Sentry SDK to the transaction
```

Comment:
Maybe write here also:

If the `transaction` is not on the scope during finish, the trace.context is wrong